### PR TITLE
fix!: rename `blocklyToolboxContents` to `blocklyToolboxCategoryGroup`

### PR DESCRIPTION
**Pull request by ruthwikchikoti**
## The basics

- [x] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change).

### Resolves

Fixes [#8344](https://github.com/google/blockly/issues/8344)

### Proposed Changes

This Pull Request renames the `blocklyToolboxContents` CSS class to `blocklyToolboxCategoryGroup`. All references to the old class name have been updated accordingly.

### CODE REVIEW

**Comment by ruthwikchikoti** [comment]
@BeksOmega have you reviewed ?!

**Comment by BeksOmega** [comment]
> @BeksOmega have you reviewed ?!

@ruthwikchikoti You didn't do the things I asked you to do yet. Once you make the requested changes I can review =)

**Comment by ruthwikchikoti** [comment]
@BeksOmega i have closed that PR this is new PR ,the branch is against  rc/v12.0.0

**Comment by BeksOmega** [comment]
@ruthwikchikoti Have you seen the [previous comment](https://github.com/google/blockly/pull/8384#issuecomment-2231734235) I left on this PR?

**Comment by ruthwikchikoti** [comment]
> @ruthwikchikoti Have you seen the [previous comment](https://github.com/google/blockly/pull/8384#issuecomment-2231734235) I left on this PR?

image
there is only one commit right ?,how can we i drop and pick,if there is only one 
i am sorry if am wrong

**Comment by ruthwikchikoti** [comment]
@BeksOmega thanks completed please review !!!!

**Review by BeksOmega — APPROVED** [review]
This looks great! Thank you for the back and forth =) Once this passes CI I'll get it merged.

**Pull request merged**
